Packs/QuestKace/Integrations/QuestKace/QuestKace.py

```diff
@@ -0,0 +1,549 @@
+
+    def tickets_list_request(self, shaping_fields: str, filter_fields: str = None) -> dict:
+        """List of Tickets.
+            Args:
+                shaping_fields: str of the shaping that will be sent in the request.
+                filter_fields: str of filter that will be sent in the request.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie
+        }
+        if filter_fields:
+            return self._http_request("GET", url_suffix=f"/service_desk/tickets?filtering={filter_fields}&"
+                                                        f"shaping={shaping_fields}", headers=headers)
+        else:
+            return self._http_request("GET", url_suffix=f"/service_desk/tickets?shaping={shaping_fields}",
+                                      headers=headers)
+
+    def create_ticket_request(self, data: str) -> dict:
+        """Create Ticket
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie,
+            'Content-Type': 'application/json'
+        }
+        return self._http_request("POST", url_suffix="/service_desk/tickets", headers=headers, data=data)
+
+    def update_ticket_request(self, ticket_id: str, data: str) -> dict:
+        """Update Ticket.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie,
+            'Content-Type': 'application/json'
+        }
+        return self._http_request("POST", url_suffix=f"/service_desk/tickets/{ticket_id}", headers=headers, data=data)
+
+    def delete_ticket_request(self, ticket_id: str) -> dict:
+        """Delete Ticket.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie,
+            'Content-Type': 'application/json'
+        }
+        return self._http_request("DELETE", url_suffix=f"/service_desk/tickets/{ticket_id}", headers=headers)
+
+
+def test_module(client: Client, args=None) -> Tuple[str, dict, dict]:
+    response = client.machines_list_request()
+    list_machines_res = response.get('Machines')
+    if list_machines_res:
+        return 'ok', {}, {}
+    else:
+        return 'Test failed', {}, {}
+
+
+def get_machines_list_command(client, args) -> Tuple[str, dict, dict]:
```

Comment:
for `get_tickets_list_command/get_assets_list_command/get_machines_list_command`
what happens if we have hundreds of items? I think we should add a `limit` arg, defaulted to 50 records.

Reply:
I added limit arg to those commands.